napari/layers/labels/labels.py

```diff
@@ -649,6 +671,9 @@ def _raw_to_displayed(self, raw):
         else:
             raise ValueError("Unsupported Color Mode")
 
+        if self.contour:
+            image = np.logical_xor(raw, ndi.binary_erosion(raw))
```

Comment:
```suggestion
            mask = np.logical_xor(raw, ndi.binary_erosion(raw))
            image[np.invert(mask)] = 0
```

Curious how you'd feel about this? Here we retain the colors of the different labels on the boundary which I think is quite nice

<img width="1156" alt="Screen Shot 2021-02-05 at 9 14 03 AM" src="https://user-images.githubusercontent.com/6531703/107066237-8fc6f900-6792-11eb-9ebc-bc45a0905a12.png">

Reply:
Love it! :heart_eyes: I had in mind to do something similar but wasn't sure yet how to do it.